source/MRMesh/MRMeshLoadStep.cpp

```diff
@@ -280,7 +280,7 @@ VoidOrErrStr readStepData( STEPControl_Reader& reader, const std::filesystem::pa
 
     const auto auxFilePath = getStepTemporaryDirectory() / "auxFile.step";
     {
-        std::ofstream ofs( auxFilePath, std::ofstream::binary );
+        std::ofstream ofs( auxFilePath );
```

Comment:
Please add a comment that stream cannot be open as binary

Reply:
Fixed.